src/main/java/medicaldoctor/entities/DoctorTable.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor.entities;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public class DoctorTable {
```

Comment:
once again, should be in another PR, but i dont think its breaking anything so fine